<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEntity</class>
 <widget class="QDialog" name="DlgEntity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entity Instance</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/16x16_ico_png/entity.png</normaloff>:/16x16_ico_png/entity.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Entity instance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_40">
        <property name="text">
         <string>Lifetime</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="chkTickEntity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tick entity</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="btnResetLifetime">
        <property name="toolTip">
         <string>Reset idle track to nothing</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/16x16_ico_png/reset.png</normaloff>:/16x16_ico_png/reset.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="chkUpdateEntity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Update entity</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="chkKillAtLifetime">
        <property name="text">
         <string>Kill at lifetime</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Idle track</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="idleAnimation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>Class Default</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="btnResetIdleAnimation">
        <property name="toolTip">
         <string>Reset idle track to nothing</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/resource.qrc">
          <normaloff>:/16x16_ico_png/reset.png</normaloff>:/16x16_ico_png/reset.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="entityLifetime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="specialValueText">
         <string>Class Default</string>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
        <property name="value">
         <double>-1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="chkKeyEvents">
        <property name="text">
         <string>Receive key events</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chkEntityLifetime">
        <property name="text">
         <string>Limit lifetime</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccept">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>idleAnimation</tabstop>
  <tabstop>btnResetIdleAnimation</tabstop>
  <tabstop>entityLifetime</tabstop>
  <tabstop>btnResetLifetime</tabstop>
  <tabstop>chkEntityLifetime</tabstop>
  <tabstop>chkKillAtLifetime</tabstop>
  <tabstop>chkUpdateEntity</tabstop>
  <tabstop>chkTickEntity</tabstop>
  <tabstop>chkKeyEvents</tabstop>
  <tabstop>btnAccept</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
